feature-libs/asm/core/facade/facade-providers.ts

```diff
@@ -0,0 +1,11 @@
+import { Provider } from '@angular/core';
+import { AsmFacadeService } from '@spartacus/asm/root';
+import { AsmQueryService } from './asm-query.service';
+
+export const facadeProviders: Provider[] = [
+  AsmQueryService,
+  {
+    provide: AsmFacadeService,
+    useExisting: AsmQueryService,
+  },
```

Comment:
The Asm facade should be implemented by the existing AsmService. 

I don't see any reason to create a separate service `AsmQueryService`

Reply:
I coordinated with Hak and Eric on their solution with Customer List and we decided this is the best course because 
Putting them together would require the ngrx methods to work with the facade proxy, which we're not sure would be side-effect free.

Reply:
There are ngrx based functions used by most of the facade services.  This is something widespread in the code base.
We need to provide users with a consistent API and I see no need to create a new Facade service to host the new method.  
Let's discuss please.

Reply:
spoke with Eric and Hak, updated code to use AsmService instead, will push shortly

Reply:
The refactoring seems to have evolved into creating a separate service and Facade pair AsmBindCartService and AsmBindCartFacade.  That's the thought process?

Reply:
The refactoring ultimately resulted in the BindCartService and BindCartFacade.  What is the motivation behind the design?  Was there perhaps too much impact to modify AsmService while avoiding breaking changes?

Reply:
We were unable to convert AsmService to a facade without inducing breaking changes elsewhere in the module to support the shift to a facade.  Additionally, when trying to use the current new facade as a regular dependency in AsmService, there were injector issues that prevented the OccAdapter from being injected at runtime.  

Creating this granular facade allowed us to begin introducing this design paradigm to the ASM module with the eventual goal of migrating AsmService to a facade in the 6.0 release, and merging any granular facades at that time.